src/main/target/common_fc_pre.h

```diff
@@ -155,6 +155,7 @@
 #define USE_TELEMETRY_HOTT
 #define USE_TELEMETRY_LTM
 #define USE_TELEMETRY_SMARTPORT
+#define USE_VARIO
```

Comment:
We should enable this for > 256 (F4 and up) only. A lot of F3 targets are often running out of space, so adding something that has little practical use is a bit pointless.

Reply:
Would it be ok to move it to >128 instead? I'm using the SPRacingF3 and @MrLAtt is using the Omnibus F3.

Reply:
Not really. If you look into these 2 targets, you will see that for both of them a lot of functionality that is important to many users has had to be removed already:

https://github.com/betaflight/betaflight/blob/74d883bfa85795d8b90af1e6900406582c8e60bc/src/main/target/OMNIBUS/target.h#L26-L50

So even if this was added to F3, it would then have to be removed again for these targets the next time flash space was getting short, since it is not an essential function for most users.